### PR TITLE
more test fixes for core featurization

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: tests/core/test_processor.py [hunk 1]
```diff
@@ -642,9 +642,16 @@ async def test_handle_message_with_session_start(
             ],
         ),
         SlotSet(entity, slot_2[entity]),
+        ActionExecuted("utter_greet"),
```
Comment (wochinge) [inline tests/core/test_processor.py:645]: @Ghostvv This seems weird. Why did it work before 🤯  It works now, but can you please double check?

Comment (Ghostvv) [reply tests/core/test_processor.py:645]: your fix seems correct to me. Could it be that session start was broken somehow?

Comment (Ghostvv) [reply tests/core/test_processor.py:645]: what policy `default_processor` has?

Comment (evgeniiaraz) [reply tests/core/test_processor.py:645]: Augmented Memo

Comment (wochinge) [reply tests/core/test_processor.py:645]: with max_history=3 🤔

Comment (wochinge) [reply tests/core/test_processor.py:645]: could it be that the `max_history` was too short before?

Comment (wochinge) [reply tests/core/test_processor.py:645]: Nope, a longer `max_history` doesn't break it on master

**Comment by wochinge** [comment]
@Ghostvv Should I merge this including the TODO or should we investigate the fixed and previously failing test?

File: tests/core/test_processor.py (hunk 1 above)
Comment (Ghostvv) [reply tests/core/test_processor.py:645]: then on master action_listen is correct because aug memo doesn't recall anything?

**Comment by Ghostvv** [comment]
yes, let's merge with TODO

**Pull request merged**